onnxruntime/contrib_ops/rocm/bert/fast_gelu_impl.cu

```diff
@@ -145,6 +145,121 @@ __global__ void FastGeluKernel4(const half2 a, const half2 b, const half2 c, con
   }
 }
 
+template <unsigned TPB>
+__global__ void FastGeluKernel8Bias(const half2 a, const half2 b, const half2 c, const half2 one2, const half2 two2,
```

Comment:
Much of the two functions FastGeluKernel8Bias and FastGeluKernel8 are the same, can we refactor the code the share the common part?